PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugRunConfiguration.java

```diff
@@ -112,6 +114,11 @@ public SparkSubmitModel getSubmitModel() {
         return getModel().getSubmitModel();
     }
 
+    @NotNull
+    public RunConfiguration createLivyConsoleRunConfiguration(SparkScalaLivyConsoleRunConfigurationFactory livyConsoleConfigFactory){
```

Comment:
Can we let the `RemoteDebugRunConfiguration` not aware about the Livy console?

Reply:
This code logic has been moved to SparkScalaLivyConsoleRunConfigurationFactory